apps/studio/src/components/menu/NewAppMenu.vue

```diff
@@ -255,9 +261,11 @@ export default {
       // Empty on purpose
     }
   },
-  mounted() {
-    this.menuBuilder = new MenuBuilder(this.$store.state.settings.settings, this.actionHandler, this.$config)
-    this.menus = this.menuBuilder.buildTemplate()
+  async mounted() {
+    if (this.settings && this.settings.length > 0) {
```

Comment:
Doesn't this mean that it's possible for the app to start without a menu? Seems like something else should happen here.